docs/dev/new_check_howto.md

```diff
@@ -8,99 +8,107 @@ aliases:
   - /guides/new_integration/
 ---
 
-To consider an Agent-based integration complete, and thus ready to be included in the core repository and bundled with the Agent package, several prerequisites must be met:
+This guide provides instructions for creating a Datadog Agent integration in the integrations-extras repo. For information on why you'd want to create an Agent integration, see [Creating your own solution][1].
 
-- A `README.md` file with the correct format and contents
-- A battery of tests verifying metrics collection
-- A `metadata.csv` file listing all of the collected metrics
-- A complete `manifest.json` file
-- If the integration collects Service Checks, the `service_checks.json` must be complete as well
+## Prerequisites
 
-These requirements are used during the code review process as a checklist. This documentation covers the requirements and implementation details for a brand new integration.
+The datadog Agent development tools require:
```

Comment:
Is this line in reference to [Developer Toolkit](https://docs.datadoghq.com/developers/integrations/new_check_howto/?tab=configurationtemplate#developer-toolkit) or the required development tools to write an agent integration? 

```suggestion
The required Datadog Agent integration development tools:
```

Reply:
This is in reference to the required development tools to write an Agent integration 👍